driver/linux/version.h

```diff
@@ -5,6 +5,6 @@
 #define _SGX_VERSION_H
 
 #define DRV_DESCRIPTION "Intel SGX DCAP Driver"
-#define DRV_VERSION "1.33"
```

Comment:
Why the version is 1.33.2, not 1.34?

Reply:
1.34 has different APIs and  this is specifically to update 1.33 as separate branch